src/GildedRose.elm

```diff
@@ -16,7 +16,7 @@ updateQuality =
 
 updateQualityItem (Item name sellIn quality) =
     let
-        quality' =
+        quality_ =
```

Comment:
That's definitely a good practice, although this is a legacy code refactoring exercise so having unclear names could be good? I don't have a strong opinion, could go either way.